test-integration/scripts/23-package.sh

```diff
@@ -28,7 +28,7 @@ if [ -f "mvnw" ]; then
     ./mvnw verify -DskipTests -P"$JHI_PROFILE"
     mv target/*.war app.war
 elif [ -f "gradlew" ]; then
-    ./gradlew bootWar -P"$JHI_PROFILE" -x test
+    ./gradlew bootWar -P"$JHI_PROFILE" -x test -x war
```

Comment:
As gradle creates not `.war.original` but `-original.war` moving `*.war` doesn't work, which means `package` fails and therefore the following e2e test. Therefore excluding the war creation here.